README.rst

```diff
@@ -183,7 +210,7 @@ like so::
     $ chmod 755 scripts/scons.py
```

Comment:
shouldn't need to chmod scons.py.. it should have executable bit set in git..?

Reply:
shrug... that's an existing bit. I can take it out.

Reply:
yup. you can remove it.  A vanilla scripts/scons.py git clone should have +x.. Do you see that?